internal/wrappers/feature-flags-http.go

```diff
@@ -64,3 +65,49 @@ func (f FeatureFlagsHTTPWrapper) GetAll() (*FeatureFlagsResponseModel, error) {
 		return nil, errors.New("failed to load feature flags for tenant")
 	}
 }
+
+func (f FeatureFlagsHTTPWrapper) GetSpecificFlag(flagName string) (*FeatureFlagResponseModel, error) {
+	accessToken, tokenError := GetAccessToken()
+	if tokenError != nil {
+		return nil, tokenError
+	}
+
+	tenantIDFromClaims, extractError := extractFromTokenClaims(accessToken, tenantIDClaimKey)
+	if extractError != nil {
+		return nil, extractError
+	}
+
+	tenantID := strings.Split(tenantIDFromClaims, "::")[1]
+	clientTimeout := viper.GetUint(commonParams.ClientTimeoutKey)
+	params := map[string]string{
+		"filter": tenantID,
+	}
+	path := fmt.Sprintf("%s/%s", f.path, flagName) // Adjusting the path to include the flag name
+
+	resp, err := SendHTTPRequestWithQueryParams(http.MethodGet, path, params, nil, clientTimeout)
+	if err != nil {
+		return nil, err
+	}
+	decoder := json.NewDecoder(resp.Body)
+
+	defer func() {
+		if err == nil {
+			_ = resp.Body.Close()
+		}
+	}()
+	switch resp.StatusCode {
+	case http.StatusBadRequest, http.StatusInternalServerError:
+		return nil, err
+	case http.StatusOK:
+		model := FeatureFlagResponseModel{}
```

Comment:
is it the same response model as the other API? are u sure it is working? cause the other response is lost of flags and here is only one flag